### PR TITLE
Some changes suggested by Ewout.

### PR DESCRIPTION
**Pull request by neunhoef**

### CODE REVIEW

File: gobench.go [hunk 1]
```diff
@@ -88,8 +87,10 @@ func doPostDocs(col driver.Collection) {
 
 	for j := 0; j < parallelism; j++ {
 		wg.Add(1)
-		go worker(times[j*nrRequestsPerWorker:(j+1)*nrRequestsPerWorker],
-			nrRequestsPerWorker)
+		go func(jj int) {
+			worker(times[jj*nrRequestsPerWorker : (jj+1)*nrRequestsPerWorker])
```
Comment (ewoutp) [inline gobench.go]: Start with the defer. Then call worker.

Comment (neunhoef) [reply gobench.go]: Done.

File: gobench.go [hunk 2]
```diff
@@ -69,8 +69,7 @@ func doPostDocs(col driver.Collection) {
 	times := make([]time.Duration, nrRequests, nrRequests)
 	wg := sync.WaitGroup{}
 
-	worker := func(times []time.Duration, nrRequestsPerWorker int) {
-		defer wg.Done()
+	worker := func(times []time.Duration) {
```
Comment (ewoutp) [inline gobench.go]: Will `times[i]` in this for loop go out of bounds?
There is no hard relation between `nrRequestsPerWorker` and `len(times)`

Comment (neunhoef) [reply gobench.go]: I believe there is: I ensure that `nrRequests` is the same as `nrRequestsPerWorker * parallelism`.
Then I create (the outer) `times` with length `nrRequests` and I hand in to the go routines slices which are disjoint but together cover the whole of `times`. Unfortunately, I call this `times` as well in the `worker`, will fix that.

**Pull request merged**